bitnami/postgresql-ha/templates/pgpool/configmap.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if and (or (.Files.Glob "files/pgpool.conf") .Values.pgpool.configuration) (not .Values.pgpool.configurationCM) }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ printf "%s-configuration" (include "postgresql-ha.pgpool" .) }}
+  labels: {{- include "postgresql-ha.labels" . | nindent 4 }}
+    app.kubernetes.io/component: pgpool
+data:
+  {{- if (.Files.Glob "files/pgpool.conf") }}
+  {{- (.Files.Glob "files/pgpool.conf").AsConfig | nindent 2 }}
+  {{- else if .Values.pgpool.configuration }}
+  pgpool.conf: |
+  {{- .Values.pgpool.configuration | nindent 4 }}
```

Comment:
Would it make sense to allow templates here? I do not know if there are settings like addresses that can be set here.

Reply:
It'd make sense. E.g. to set properties like the ones below:
```
backend_hostname0 = 'postgresql-ha-postgresql-0.postgresql-ha-postgresql-headless.default.svc.cluster.local'
```
Let me address this